feature/git/dffml_feature_git/feature/operations.py

```diff
@@ -177,6 +177,32 @@
             'commit': sha
             }
 
+@op(inputs={
```

Comment:
This needs to move into `examples/shouldi/shouldi/bandit.py`

Reply:
Change from `subprocess.check_output` to `asyncio.create_subprocess_exec` as shown in `examples/shouldi/shouldi/safety.py`